metomi/rose/apps/rose_ana.py

```diff
@@ -34,16 +34,17 @@
 from contextlib import contextmanager
 
 # Rose modules
-from rose import TYPE_LOGICAL_VALUE_TRUE
-from rose.reporter import Reporter
-from rose.resource import ResourceLocator
-from rose.app_run import BuiltinApp
-from rose.env import env_var_process
+from metomi.rose import TYPE_LOGICAL_VALUE_TRUE
+from metomi.rose.reporter import Reporter
+from metomi.rose.resource import ResourceLocator
+from metomi.rose.app_run import BuiltinApp
+from metomi.rose.env import env_var_process
 
 
 class KGODatabase(object):
     """
-    KGO Database object, stores comparison information for rose_ana apps.
+    KGO Database object, stores comparison information for metomi.rose_ana
```

Comment:
Frustratingly a lot of this diff is to prevent to addition of `metomi.` from making `len(lines) > 79`.

Reply:
@kinow  - is that a confused face or a sad face?
if confused:
```python
# If a line is > 72 chars long...
from rose.verylongname import AReallyQuiteExtraordinarilyLongClassName

# ... metomi. will push it over 79 chars...
from metomi.rose.verylongname import AReallyQuiteExtraordinarilyLongClassName

# ... so will need a bit of refactoring/jiggery-pokery
from metomi.rose.verylongname import (
    AReallyQuiteExtraordinarilyLongClassName
)
```

Reply:
`AReallyQuiteExtraordinarilyLongClassName`, rename the class!

Reply:
Also it isn't `metomi.rose_ana` it's just `rose_ana` or "Rose Ana".

Reply:
> `AReallyQuiteExtraordinarilyLongClassName`, rename the class!

This was a made-up example to illustrate the problem. Your solution is of course correct and the class should be re-named. The point still stands though that the addition of metomi. requires a change where the magnitude of the diff is very much greater than the significance of the change.